src/Formatter/AbstractFormatter.php

```diff
@@ -48,33 +34,4 @@ public static function accepts()
      * @return string
      */
     abstract protected function body(PayloadInterface $payload);
-
-    /**
-     * Get the response status from the payload.
-     *
-     * @param PayloadInterface $payload
-     *
-     * @return integer
-     */
-    public function status(PayloadInterface $payload)
```

Comment:
@shadowhand Even knowing about this feature I keep thinking how-to modify or to traverse it.. But with the interface I read your message. I just don't understand why not make this method public?

I like the way it's done now. I was referring to [the Scrutinizer CI metrics](https://scrutinizer-ci.com/g/equip/framework/code-structure/master/class/Equip%5CResponder%5CFormattedResponder).